pkgs/applications/science/math/lp_solve/default.nix

```diff
@@ -37,6 +37,7 @@ stdenv.mkDerivation rec {
     license     = licenses.gpl2Plus;
     maintainers = with maintainers; [ smironov ];
     platforms   = platforms.unix;
+    broken = false;
```

Comment:
I think you meant `broken = true`. Can you also align the `=` with the rest for concistency's sake ?